hnn_core/tests/test_drives.py

```diff
@@ -115,10 +116,8 @@ def test_add_drives():
     net = Network(params, legacy_mode=False)
 
     # Ensure weights and delays are updated
-    weights_ampa = {'L2_basket': 1.0, 'L2_pyramidal': 3.0,
-                    'L5_basket': 2.0, 'L5_pyramidal': 4.0}
-    syn_delays = {'L2_basket': 1.0, 'L2_pyramidal': 2.0,
-                  'L5_basket': 3.0, 'L5_pyramidal': 4.0}
+    weights_ampa = {'L2_basket': 1.0, 'L2_pyramidal': 3.0, 'L5_pyramidal': 4.0}
```

Comment:
Removed to check `weight=None` conditon?

Reply:
Are you referring to the `L5_basket` weight? I removed this because most of the tests in this function create distal drives.